Formula/canu.rb

```diff
@@ -11,7 +11,6 @@ class Canu < Formula
 
   def install
     system "make", "-C", "src"
-    arch = Pathname.new(Dir["*/bin"][0]).dirname
```

Comment:
I think you misinterpreted my previous comment. I only meant delete the last line. Sorry for the confusion
Delete this:
```ruby
    doc.install Dir["README.*"]
```
Keep this:
```ruby
    arch = Pathname.new(Dir["*/bin"][0]).dirname
    rm_r arch/"obj"
    prefix.install arch
    bin.install_symlink prefix/arch/"bin/canu"
```